scripts/trustbloc_local_setup.sh

```diff
@@ -34,6 +34,7 @@ cat > ${SANDBOX_HOME}/hosts <<EOF
 127.0.0.1 org3.trustbloc.local
 127.0.0.1 testnet.trustbloc.local
 127.0.0.1 issuer-adapter.trustbloc.local
+127.0.0.1 rp.trustbloc.local
```

Comment:
This entry is already there - https://github.com/trustbloc/edge-sandbox/pull/411/files#diff-3c99bcef25e7c063b6c312ac0ea6e068L15

Reply:
@rolsonquadras fixed